src/tmt_web/api.py

```diff
@@ -113,6 +220,7 @@ def find_test(
 
     # Disable Celery if not needed
     if os.environ.get("USE_CELERY") == "false":
+        logger.debug("Celery disabled, processing request synchronously")
         response_by_output = {
             "html": HTMLResponse,
             "json": JSONResponse,
```

Comment:
When testing JSON response **locally without celery** (see the README.md), the response is returning JSON as a **string** instead of proper JSON:
- http://0.0.0.0:8000/?test-url=https://github.com/teemtee/tmt&test-name=/tests/core/smoke&format=json

The problem is the json_generator uses `pydantic.BaseModel.model_dump_json()`. We should use `model_dump()` instead or convert the input into the JSONResponse to `dict`.

It seems to be working fine with this quick hack:

```
return response(content=json.loads(service.main(**service_args))) 
```

Reply:
Cheers, your comment actually made it all 'click' for me and I finally understand it all :). anyway, should be fixed :)